### PR TITLE
CXEC-21318: fix breadcrumbs

### PR DESCRIPTION
**Pull request by Parthlakhani**

### CODE REVIEW

File: feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts [hunk 1]
```diff
@@ -66,4 +77,26 @@ export class CustomerTicketingPageMetaResolver
...
+      this.basePageMetaResolver.resolveBreadcrumbs(),
+    ]).pipe(
+      map(([customeServiceBreadCrumb, breadcrumbs = []]) => {
+        const [home, ...restBreadcrumbs] = breadcrumbs;
+        return [home, ...customeServiceBreadCrumb, ...restBreadcrumbs];
+      })
+    );
+  }
+
+  protected customeServiceBreadCrumb$: Observable<BreadcrumbMeta[]> =
```
Comment (adolfobee) [inline feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts]: customerServiceBreadCrumb

File: feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts [hunk 2]
```diff
@@ -66,4 +77,26 @@ export class CustomerTicketingPageMetaResolver
   resolveRobots(): Observable<PageRobotsMeta[]> {
     return this.basePageMetaResolver.resolveRobots();
   }
+
+  resolveBreadcrumbs(): Observable<BreadcrumbMeta[]> {
+    return combineLatest([
+      this.customeServiceBreadCrumb$,
```
Comment (adolfobee) [inline feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts]: customerServiceBreadCrumb

File: feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts [hunk 3]
```diff
@@ -66,4 +77,26 @@ export class CustomerTicketingPageMetaResolver
...
+      this.basePageMetaResolver.resolveBreadcrumbs(),
+    ]).pipe(
+      map(([customeServiceBreadCrumb, breadcrumbs = []]) => {
+        const [home, ...restBreadcrumbs] = breadcrumbs;
+        return [home, ...customeServiceBreadCrumb, ...restBreadcrumbs];
```
Comment (adolfobee) [inline feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts]: customerServiceBreadCrumb

File: feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts [hunk 4]
```diff
@@ -66,4 +77,26 @@ export class CustomerTicketingPageMetaResolver
   resolveRobots(): Observable<PageRobotsMeta[]> {
     return this.basePageMetaResolver.resolveRobots();
   }
+
+  resolveBreadcrumbs(): Observable<BreadcrumbMeta[]> {
+    return combineLatest([
+      this.customeServiceBreadCrumb$,
+      this.basePageMetaResolver.resolveBreadcrumbs(),
+    ]).pipe(
+      map(([customeServiceBreadCrumb, breadcrumbs = []]) => {
```
Comment (adolfobee) [inline feature-libs/customer-ticketing/core/services/customer-ticketing-page-meta.resolver.ts]: customerServiceBreadCrumb

**Review by khristian23 — APPROVED** [review]

**Review by manpreetlamba4 — APPROVED** [review]

**Pull request merged**